.github/workflows/pull-request-opened.yml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Automate pull requests triage
+
+name: Pull request opened
+
+on:
+  pull_request:
```

Comment:
it won't work for forks as forks won't have access to secrets with "pull_request" event